src/Components/Components/src/Rendering/RenderTreeBuilder.cs

```diff
@@ -794,7 +794,8 @@ internal void TrackAttributeName(string name)
             seenAttributeNames[name] = _entries.Count; // See comment in ProcessAttributes for why this is OK.
         }
 
-        void IDisposable.Dispose()
+        /// <inheritdoc />
+        public void Dispose()
```

Comment:
Do we know why this was doing a private interface implementation?

/cc: @pranavkm

Reply:
I couldn't think of a good reason for it, and in fact it seems pretty dangerous because it makes it harder for people to realise the type is `IDisposable`. As such it seems like a clear improvement to make the disposability more obvious.

Reply:
Yeah, this just seems like a compelling change. I'll shoot an email to API review as an FYI